File: parse.js
var request = require('request');
var fs = require('fs');

var ORG = 'glib';
var ID = '123456';

var base_url = 'http://35.187.158.185:8080/parse';

var url = [base_url, ORG, ID].join('/');

var config = {};
config.url = url;
config.headers = {'Content-Type': 'application/json'};
config.json = {"query": "i want to eat a burger"};

request.post(config, function (err, resp, body) {
    if (err) {
        console.log('Error!');
    } else {
        if (resp.statusCode === 200) {
            console.log('Query: ', body.query);
            console.log('Intent: ', body.intent);
            console.log('Score: ', body.score);
            console.log('Entities: ', body.context);
        } else {
            console.log('Error: ' + body.message);
        }

    }
});

File: parse.py
import requests
import json

ORG = 'glib-sample'
ID = '123456'

base_url = 'http://35.187.158.185:8080/parse'
url = '/'.join([base_url, ORG, ID])
headers = {'content-type': 'application/json'}

data = {"query": "i want to eat a burger"}

r = requests.post(url, data=json.dumps(data), headers=headers)
if r.status_code == 200:
    response = json.loads(r.content)
    print 'Query: ', response['query']
    print 'Intent: ', response['intent']
    print 'Score: ', response['score']
    print 'Entities: ', response['context']
else:
    response = json.loads(r.content)
    print 'Error: ', response['message']
